blog/_posts/2017-02-22-repl-0.6-highlights.md

```diff
@@ -0,0 +1,94 @@
+
+Unfortunately, for technical reasons (not supporting bracket pasting), this feature does not work on the standard cmd-terminal in windows. There are however multiple alternative terminal emulators on Windows that do support this, e.g. [Console2](https://sourceforge.net/projects/console/), [Cygwin](https://www.cygwin.com/) and [Cmder](http://cmder.net/).
+
+## Printing of stacktraces
+
+The accuracy of the stacktraces getting printed when an error is thrown is improved in every release. Function calls are now even tracked through inlining so that often the full chain of calls up to the error is showed.
+This, in combination with the common way of writing Julia code by which one frequently write functions that does a small amount of job and then dispatch to another function makes for quite deep stacktraces.
+In addition, types in Julia can easily become long to print since it is so easy to make types taking many type parameters (see for example `StridedArray`).
+The conclusion is that stacktraces in Julia tend to both be deep with each stackframe needing a significant amount of space.
+
+Previously, stacktraces were being printed without much structure. Everything was printed in bold red which made it a bit difficult to find the information one was looking. As an example, here is a screenshot from a thrown error on 0.5:
+
+![Stacktrace in 0.5](https://github.com/JuliaLang/julialang.github.com/blob/master/blog/_posts/repl0.6/0.5stacktrace.png)
+
+Finding the relevant information can be quite difficult, espeically the file name and line numbers since they are not aligned in any way. In 0.6, this is what is currently showed for the same error:
+
+![Stacktrace in 0.6](https://github.com/JuliaLang/julialang.github.com/blob/master/blog/_posts/repl0.6/0.6stacktrace.png)
+
+We got rid of printing the contents of the `Array` on a `BoundsError` but more relevant for this section, the way stacktrace is showed is significantly different. It is no longer written in bold red, which helps distinguish it from the error message. The parts that are most likely to be interesting (the function name and the line info) are now bold and every stackframe is numbered which makes them easier to refer to. Furthermore, it is also possible to customize the colors that the functions and line infos are printed by setting the environment variables `JULIA_STACKFRAME_FUNCTION_COLOR` and `JULIA_STACKFRAME_LINEINFO_COLOR` respectively. Executing:
+
+```jl
+julia> ENV["JULIA_STACKFRAME_LINEINFO_COLOR"] = :cyan;
+
+julia> ENV["JULIA_STACKFRAME_FUNCTION_COLOR"] = :yellow;
+```
+
+and rerunning the example, now displays:
+
+![Stacktrace in 0.5 with configured colors](https://github.com/JuliaLang/julialang.github.com/blob/master/blog/_posts/repl0.6/0.6stacktrace_conf.png)
+
+In addition to the new visual look, there is also a new feature, possible now that we can refer to stackframes by number.
```

Comment:
"Visual look" is redundant since (presumably) all looks are visual. :) Could just be "look."